integrationtests/tests/fixtures/mod.rs

```diff
@@ -114,31 +124,37 @@ pub struct Fixtures {
 pub async fn test<B>(
     num_peers: u16,
     f: impl FnOnce(
-        FederationTest,
-        UserTest<UserClientConfig>,
-        Box<dyn BitcoinTest>,
-        GatewayTest,
-        Box<dyn LightningTest>,
-    ) -> B,
+            FederationTest,
+            UserTest<UserClientConfig>,
+            Box<dyn BitcoinTest>,
+            GatewayTest,
+            Box<dyn LightningTest>,
+        ) -> B
+        + std::marker::Copy,
 ) -> anyhow::Result<()>
 where
     B: Future<Output = ()>,
 {
-    let fixtures = fixtures(num_peers).await?;
-    f(
-        fixtures.fed,
-        fixtures.user,
-        fixtures.bitcoin,
-        fixtures.gateway,
-        fixtures.lightning,
-    )
-    .await;
-    fixtures.task_group.shutdown_join_all(None).await
+    let nodes = vec![GatewayNode::Cln, GatewayNode::Lnd];
```

Comment:
This just runs each integration test once with LND, once with CLN. I should parameterize this better so that non-lightning tests don't run twice ...

Reply:
Maybe this array should be passed in by the test itself? Otherwise we're going to need some mechanism for determining which test is and isnt a lightning test

Reply:
Relevant to https://github.com/fedimint/fedimint/pull/1702

Reply:
wontfix for now

Reply:
I updated this